js/hw19.js

```diff
@@ -0,0 +1,54 @@
+class Human {
+  constructor(name, gender) {
+    this.name = name;
+    this.gender = gender;
+  }
+}
+
+class Flat {
+  residentsArray = [];
+  addResidents(resident) {
```

Comment:
```suggestion
  addResident(resident) {
```
 Ви все ж таки одаєте мешканців по одному
 якщо залишити множину, тоді метод напевно повинен був би приймати масив людей